sdk/webpubsub/Azure.Messaging.WebPubSub/CHANGELOG.md

```diff
@@ -1,14 +1,9 @@
 # Release History
 
-## 1.1.0-beta.1 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
+## 1.1.0 (2022-08-03)
```

Comment:
CI is complaining because this value has to match the version in the project file.   We can either revert this to the beta and update right before release, or update both now.

Reply:
Thanks for taking a look into it Jesse, I am also updating the project file to 1.1.0. Do you know who could be taking care of releasing the 1.1.0 SDK?

Reply:
As package owners, your team also has responsibility for their release.   If you're unfamiliar with how to do so, feel free to reach out and I'd be happy to share the internal documentation on the process.

Reply:
Thanks, I think I have the permission to trigger the release pipeline but need approval for releasing the packages. Might need your help there. Also, I updated the version back to beta.1 considering we have more changes on the way for 1.1.0